### PR TITLE
Fix documentation builds for RTD

### PR DESCRIPTION
**Pull request by mossblaser**
Removed broken docs and setup fallback mock for when numpy inevitably doesn't install.

Live demo: http://rig.readthedocs.org/en/fix_docs/index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +38,7 @@ SpiNNaker.  Tools exist for:
  - Clock discipline
  - Routing table minimisation
  - Optimal routing key allocation
+
+# Documentation
```
Comment (mundya) [inline README.md]: Worth mentioning how to build the docs?

File: docs/source/conf.py [hunk 2]
```diff
@@ -1,4 +1,18 @@
 # -*- coding: utf-8 -*-
+
+# Allow the docs to be built by ReadTheDocs without building numpy.
```
Comment (mundya) [inline docs/source/conf.py]: What a faff, I can't believe this isn't even provided by RTD.

**Comment by mundya** [comment]
LGTM.

File: docs/source/conf.py (hunk 2 above)
Comment (mossblaser) [inline docs/source/conf.py]: Yup... especially since they host the numpy docs!!

File: README.md (hunk 1 above)
Comment (mossblaser) [inline README.md]: Will attempt to do this and also incorporate the changes from the development-docs branch in a PR coming soon to this repository.

**Comment by mossblaser** [comment]
Thanks very much!

File: docs/source/conf.py (hunk 2 above)
Comment (mundya) [inline docs/source/conf.py]: But even the "Mock" thing, they go to the lengths of providing the code!

**Pull request merged**